ethminer/MinerAux.h

```diff
@@ -815,15 +900,16 @@ class MinerCLI
 
 		PoolManager mgr(client, f, m_minerType);
 		mgr.setReconnectTries(m_maxFarmRetries);
-		mgr.addConnection(m_farmURL, m_port, m_user, m_pass);
-		if (!m_farmFailOverURL.empty()) {
-			if (!m_fuser.empty())
-				mgr.addConnection(m_farmFailOverURL, m_fport, m_fuser, m_fpass);
-			else
-				mgr.addConnection(m_farmFailOverURL, m_fport, m_user, m_pass);
+		mgr.addConnection(m_endpoints[k_primary_ep_ix]);
```

Comment:
hmm.. k_max_endpoints is set to 6, but you only add endpoints 0 and 1 ?
Doesn't it make sense to just iterate m_endpoints and add valid ones?

Reply:
That would be nice. But, for example, the -SF could be specified before -S and would therefore be added in the wrong order. That's the only reason I chose a fixed size array, so that I could access it using an index of 0 or 1 for compatible support of legacy.
For -P parameters, you can have up to 6. When we drop support for legacy parameters the we can move to a more suitable connection list container.

Reply:
Ah, I misunderstood what you meant. You mean once all parameters are parsed. Yes, good idea.